requirements.txt

```diff
@@ -1,13 +1,12 @@
 requests==2.26.0
 --extra-index-url https://www.piwheels.org/simple
-opencv-python==4.5.4.58 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version == "3.10"
-opencv-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version != "3.10"
 opencv-contrib-python==4.5.4.58 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version == "3.10"
-opencv-contrib-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version != "3.10"
-opencv-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
-opencv-contrib-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
--e ./depthai_sdk
+opencv-contrib-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and platform_machine != "arm64" and python_version != "3.10"
+opencv-contrib-python>=4.6.0.66 ; platform_machine == "arm64"  # MacOS
```

Comment:
Please avoid upgrading to 4.6 because of the issue with charuco calibration board 
https://github.com/opencv/opencv_contrib/issues/3291

Mentioning this because this repo contains `calibrate.py` that will break if the version get updated to 4.6

Reply:
Thanks, we'll downgrade until resolved!